harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -964,11 +962,10 @@ def _validate(self, searcher_op: Optional[core.SearcherOperation] = None) -> Dic
             batch_metrics = []
 
             assert isinstance(self.validation_loader, torch.utils.data.DataLoader)
-            if len(self.validation_loader) == 0:
-                raise RuntimeError("validation_loader is empty.")
             for callback in self.callbacks.values():
                 callback.on_validation_epoch_start()
 
+            idx = -1
```

Comment:
You are taking a check which was fully contained in two consecutive lines and spreading it over a wider area here.  I don't want you to rewrite it because what you have is simple and easy, but can you just put a comment above `idx = -1` to explain why -1 is significant?

Reply:
That's a nice idea. Done.